contracts/StakingBank.sol

```diff
@@ -1,78 +1,102 @@
-pragma solidity ^0.4.24;
-
-import 'zeppelin-solidity/contracts/ownership/Ownable.sol';
-import 'zeppelin-solidity/contracts/MerkleProof.sol';
-import "zeppelin-solidity/contracts/ReentrancyGuard.sol";
-import 'token-sale-contracts/contracts/Token.sol';
-import 'token-sale-contracts/contracts/HumanStandardToken.sol';
-
-contract StakingBank is Ownable, ReentrancyGuard {
-  address public tokenAddress;
-  address public chainAddress;
-
-  constructor(
-    address _tokenAddress,
-    address _chainAddress
-  ) public {
-    tokenAddress = _tokenAddress;
-    chainAddress = _chainAddress;
-  }
+pragma solidity 0.5.0;
 
-  mapping(address => uint256) public balances;
-  mapping(address => bool) public locks;
+import "./interfaces/IStakingBank.sol";
+import "./StakingBankStorage.sol";
 
-  function receiveApproval(
-    address _from,
-    uint256 _value,
-    address _token,
-    bytes _data
-  ) public nonReentrant returns (bool success) {
-    require(_token == tokenAddress);
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
-    Token token = Token(tokenAddress);
-    uint256 allowance = token.allowance(_from, this);
+import "contract-registry/contracts/interfaces/RegistrableWithSingleStorage.sol";
+import "contract-registry/contracts/storageStrategy/interfaces/IStorageStrategy.sol";
+import "digivice/contracts/interfaces/IVerifierRegistry.sol";
+import "andromeda/contracts/interface/IChain.sol";
```

Comment:
Why does `andromeda/contracts/interface` vs `digivice/contracts/interfaces`.

It's important to be *consistent*. Use `interfaces`